letsencrypt/client.go

```diff
@@ -13,11 +13,12 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	lego "github.com/xenolf/lego/acme"
+	loge "github.com/xenolf/lego/log"
 )
 
 const (
 	StorageDir       = "/etc/letsencrypt"
-	ProductionApiUri = "https://acme-v01.api.letsencrypt.org/directory"
+	ProductionApiUri = "https://acme-v02.api.letsencrypt.org/directory"
 	StagingApiUri    = "https://acme-staging.api.letsencrypt.org/directory"
```

Comment:
This also needs to be updated to the v02 endpoint `https://acme-staging-v02.api.letsencrypt.org/directory`, otherwise, we get the error:

> 03/07/2018 13:50:27time="2018-07-03T12:50:27Z" level=fatal msg="LetsEncrypt client: Could not create client: directory missing new registration URL"

Reply:
Thanks. Fixed